code/modules/mob/living/carbon/life.dm

```diff
@@ -21,6 +21,7 @@
 
 	//Updates the number of stored chemicals for powers
 	handle_changeling()
+	handle_devil()

```

Comment:
Soul power should only change upon a contract being signed, upon a lawyer/hop returning someone's soul, and upon resurrection.